tests/server/api/api_test.go

```diff
@@ -969,13 +969,20 @@ func TestPreparingProgress(t *testing.T) {
 			StartTimestamp: time.Now().UnixNano() - 100,
 		},
 	}
-
-	for _, store := range stores {
+	// store 4 and store 5 are preparing state while store 1, store 2 and store 3 are state serving state
+	for _, store := range stores[:2] {
```

Comment:
Don't we need to add store1,2,3 which never seem to be added?

Reply:
stores[:2] is for 1/2/3 and stores[2:] is for 4/5